plans/academic/interview_preparedness.markdown

```diff
@@ -0,0 +1,153 @@
+# Critical Challenges Planning
+
+## The Challenge
+
+* What Critical Challenge are you going to work on?
+
+We need students to be well-prepared for the technical challenges and code reviews in their job hunting process.
+
+* Why is this a “challenge”?
+
+We have seen numerous examples of otherwise successful or talented
+students "freezing up" or otherwise underperforming in focused technical
+interview peparation. This has been especially frustrating in many cases since it
+doesn't demonstrate an accurate representation of the students'
+abilities.
+
+* Can it be solved? What would be value to the organization/team in solving it or working towards a solution?
+
+Improving student technical interview performance would likely give us
+might higher job yields overall. It often takes a lot of effort to "open
+the door" on a job opportunity (getting past initial gatekeepers, etc),
+so it's particularly damaging when students get through these early
+hurdles only to be stopped by a technical interview.
+
+* What happens if we fail to solve it / improve it?
+
+Students will continue to struggle with the job search. Some might be
+unable to find jobs, and many will end up with less exciting job
+opportunities than they could have had.
+
+* What skills do you have that make you the right person to work on this challenge?
+
+- interviewed lots of eng candidates at livingsocial
+- have been running DSA sessions since being at Turing, focusing on
+  more CS-heavy topics that often appear in interviews
+
+## The Past
+
+* What’s been done so far?
+
+- DSA as ongoing elective
+- Monday Technical Interview Prep sessions over recent months
+- Moving module 3 and 4 assessments to more intensive solo technical
+  projects
+- Adding regular long code challenges in module 4 to get students more
+  accustomed to the time-crunch environment
+
+* Was it effective? How do you know?
+
+Effective in some cases. Have heard instances of students
+being interviewed on topics covered in DSA or TIP. But not yet the
+universal reach we would like to see.
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+Has been a general group challenge, and especially Jeff's with the TIP
+work. Jeff identified the original problem based on student feedback
+from job interviews.
+
+## The Present
+
+* What are the first steps that you’re ready to execute now?
+
+- Turn DSA into more of a test/problem-solving session? (less free-form,
+  more structured)?
+- Start having regular 1-on-1 practice interviews with module 3 and 4
+  students (how to schedule this? where to get the time?)
```

Comment:
What about "mock-interview" nights twice per module where current mentors volunteer to do practice interviews either in person or remotely with students?